walkman.el

```diff
@@ -134,18 +134,37 @@ INSECURE is optional flag to make insecure request."
     (re-search-forward (format "^ *%s \\(.*\\)$" walkman--verb-regexp))
     (match-string 2)))
 
+(defun walkman--format-headers (k v with-quote form)
+  "Format the headers with key K and value V.
+
+WITH-QUOTE indicates if this needs to be quoted.
+FORM indicates if this is a form -F header."
+  (let ((value
+         (if (string-match "\\[\\[\\([^\]]+\\)\\]\\]" v) ;; org file link
+             (concat "@" (replace-regexp-in-string "\\[\\|\\]" "" v))
+           v)))
+    (cond ((and with-quote form) (format "'%s=%s'" k value))
+          (with-quote (format "'%s: %s'" k value))
+          (form (format "%s=%s" k value))
+          (t (format "%s: %s" k value)))))
+
 (defun walkman--extract-headers (&optional with-quote)
   "Extract HTTP headers.
 
 WITH-QUOTE indicates that header values need to be quoted."
   (save-excursion
     (goto-char (point-min))
-    (let ((headers '()))
-      (while (re-search-forward "^ *- \\(.+:.+\\)$" (point-max) t)
-        (if with-quote
-            (push (format "'%s'" (match-string 1)) headers)
-          (push (match-string 1) headers))
-        (push "-H" headers))
+    (let ((headers '())
+          (form nil))
+      (while (re-search-forward "^[ \t]*\\(:form:\\|- \\([^:]+\\):[ \t]*\\(.+\\)\\)$" (point-max) t)
+        (if (equal (match-string-no-properties 1) ":form:")
```

Comment:
May I suggest using `:FORM:` here? That way we're following the Org Drawer convention (such as what we see with `:PROPERTIES:`. What do you think?

Reply:
Sounds good! Will update. As you can tell, I'm not super familiar with org drawers, so thank you for your suggestion.